apps/showcase/src/components/showcase/configuration/configuration-pres.config.ts

```diff
@@ -45,14 +45,14 @@ export interface ConfigurationPresConfig extends Configuration {
   shouldProposeRoundTrip: boolean;
 }
 
-export const CONFIGURATION_PRES_DEFAULT_CONFIG: ConfigurationPresConfig = {
+export const CONFIGURATION_PRES_DEFAULT_CONFIG: Readonly<ConfigurationPresConfig> = {
```

Comment:
Shouldn't we adapt the generator as well ? (maybe a ng update)

Reply:
Could be done in another PR but at least we should open an issue to not forget it

Reply:
Issue opened: #2530